### ISSUE TITLE
Have different predefined packages for different uses

### ISSUE THREAD

**Issue by karan-dreambits** [OxideApps/oxide#7]
The idea here is that we can put different type of packages as per end goal of the user downloading this distribution.

For e.g. If my end goal is to have a ecommerce site then ideal package would have sales, purchase, ecommerce and payment modules along with oca's top useful modules.

Finding relevant oca addons is a lengthy process and requires lot of testing so having such package where approved extra-features modules come preinstalled would enhance the final image of odoo to the newcomers and clients who might not know about the possibilities with odoo.

**Comment by leemannd** [comment]
@karan-dreambits It's a good idea. But my first input is to target base modules that are really usefull. Like having `base_technical_features`, and so on.

**Comment by karan-dreambits** [comment]
@leemannd I agree with your view. Having the best and most useful modules preinstalled for all would be the best thing. 
We can label this idea as "good-to-have" or "future-feature" if enough people like it

**Comment by dreispt** [comment]
The goal here should be to have a "core" distribution, that can then be used as a base for downstream distributions focusing on specific use cases or verticals.
This make me think that this core distribution should be unbranded, and let that branding be done downstream.
